core/lib/protobuf_config/src/chain.rs

```diff
@@ -165,3 +166,60 @@ impl ProtoRepr for proto::Mempool {
         }
     }
 }
+
+impl ProtoRepr for proto::NetworkConfig {
+    type Type = configs::chain::NetworkConfig;
+    fn read(&self) -> anyhow::Result<Self::Type> {
+        Ok(Self::Type {
+            network: required(&self.network)
+                .and_then(|x| Ok(proto::Network::try_from(*x)?))
+                .context("network")?
+                .parse(),
+            zksync_network: required(&self.zksync_network)
+                .context("zksync_network")?
+                .clone(),
+            zksync_network_id: L2ChainId::try_from(
+                *required(&self.zksync_network_id).context("zksync_network_id")?,
+            )
+            .map_err(|e| anyhow!(e))?,
+        })
+    }
+
+    fn build(this: &Self::Type) -> Self {
+        Self {
+            network: Some(proto::Network::new(&this.network).into()),
+            zksync_network: Some(this.zksync_network.clone()),
+            zksync_network_id: Some(this.zksync_network_id.as_u64()),
+        }
+    }
+}
```

Comment:
Are those the same values from genesis? If so we should deprecate the `NetworkConfig` struct.

Reply:
yep. 
no problem

Reply:
network is `l1_chain_id`
`zksync_network` and `zksync_network_id` are equivalent and we could use `l2_chain_id`

Reply:
Changed, although I set `zksync_network` as `zksync_network_id.to_string()` because I couldn't find where else to get this information from. Would you like me to add it to any specific config file?